builder/dockerfile/containerbackend.go

```diff
@@ -0,0 +1,156 @@
+package dockerfile
```

Comment:
For the logging. The `Run()` method should just return the context cancellation error.